### PR TITLE
Discovery in-memory / WSS tests

### PR DESCRIPTION
**Pull request by dymayday**
## PR summary

This PR proposes to add tests in order to make sure discovery / bootstrap connect works for both memory transport and wss transport.

Reference https://github.com/holochain/lib3h/issues/399

This work is irrelevant now because its fated to be discarded when ghost_actor v2 will kick off.

## Review checklist 
- [ ] The story has unit or integration tests
- [ ] No new bugs, and any tech-debt is identified and justified
- [ ] There is enough API documentation (how to use)
- [ ] There is enough code documentation (how the code works)

### CODE REVIEW

**Comment by zippy** [comment]
I remembered what I did for discovery in the memory transport.  Look at try_discover()  This gets called in process_concrete() of GhostMemoryTransport as periodic discovery.  See line 183 of ghost_transport_memory.   So we need to do something similar for Websockets.
In the case of the in-memory transport we do this until some other node is found as bootstrapping, assuming that all further nodes will be found through gossip.  The same assumption is reasonable for websockets.
Thus @meidhy you only need to add something similar for websockets, not any discover call in the ghost engine constructor like I said earlier.

File: crates/lib3h/src/transport/websocket/actor.rs [hunk 1]
```diff
@@ -338,6 +345,64 @@ impl GhostTransportWebsocket {
...
     }
+
+    /// Try to discover peers on the network using mDNS.
+    fn try_discover(&mut self) {
+        self.last_discover = match self.last_discover {
+            None => Some(Instant::now()),
+            Some(last_discover) => {
+                // Let's check if it time to discover some peers, but only if we already
+                // did some url binding
+                if self.bound_url.is_none()
```
Comment (zippy) [inline crates/lib3h/src/transport/websocket/actor.rs]: Doesn't this have to be `.is_some()`?

Comment (dymayday) [reply crates/lib3h/src/transport/websocket/actor.rs]: Oups, good catch !!

**Comment by dymayday** [comment]
This work is irrelevant now because its fated to be discarded when ghost_actor v2 will kick off.

So we can merge it as soon as the CI pass IMO.

**Pull request merged**